src/style.css

```diff
@@ -0,0 +1,78 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  font-family: Arial, serif;
+  color: #fff;
+  font-size: 100px;
+  height: 300px;
+  min-width: 300px;
+  width: 33%;
+
```

Comment:
remove empty line

Reply:
1. it's better not to use an empty lines to visually divide, in our case, flexbox rules from others?
2. I should re-push files after fixes?